float64[] waypointsx
float64[] waypointsy
float64[] waypointsz
float64[] waypointsid
float64[] startpva
float64[] endpva
float64 flyspeed
string mode
bool request
---
bool success
string wrong_code
float64[] ret_coefficients
int64 ret_n_poly
int64 ret_n_order
float64[] ret_wpx
float64[] ret_wpy
float64[] ret_wpz
float64[] ret_wpvx
float64[] ret_wpvy
float64[] ret_wpvz
float64[] ret_wpax
float64[] ret_wpay
float64[] ret_wpaz
int64[] ret_id